tensorflow/python/data/experimental/kernel_tests/parse_example_dataset_test.py

```diff
@@ -145,7 +147,8 @@ def testEmptySerializedWithAllDefaults(self):
         expected_values=expected_output,
         create_iterator_twice=True)
 
-  @test_util.run_deprecated_v1
+  @combinations.generate(
+      combinations.combine(tf_api_version=[1], mode=["graph", "eager"]))
```

Comment:
`run_deprecated_v1` should be graph v1 + graph v2 not graph v1 + eager v1. So replace it with `test_base.graph_only_combinations()` throughout this PR.

Reply:
Oh, my misunderstanding. Changed them now.